Makefile

```diff
@@ -120,3 +122,13 @@ docker-shell:
 check:
 	clear
 	@docker-compose run --rm web python manage.py check
+
+make-messages:
+	clear
+	@docker-compose run --rm web python manage.py makemessages -l en
+	@docker-compose run --rm web python manage.py makemessages -l af
```

Comment:
Such a list will become unwieldy quickly. Does `--all` work well?

Reply:
I tried --all, does not seem to generate anything